selfdrive/car/toyota/fingerprints.py

```diff
@@ -824,6 +824,7 @@
       b'\x03896634760100\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701004\x00\x00\x00\x00',
+      b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00????????????????',
```

Comment:
@sshane would be nice to have a "does this look like the other versions" check in the bot